stable/external-dns/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description:
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.2.0
+version: 1.2.1
```

Comment:
I think at least this should be 1.3.0 as enabling the cloudflare options changes things quite a bit.
Either that or leave it false for opt-in.

Reply:
For me it seems logical that once anyone opts in for Cloudflare they want to use it too, not just the dns part but all the additional stuff. In my perspective this option was missing accidentally since this should be the normal behaviour.

With [this](https://github.com/kubernetes-incubator/external-dns/pull/650) coming, there will be an annotation based setting per ingress.

Because of this I would pick the 1.3.0 bump